### DISCUSSION TITLE
LAN mode not working anymore

### DISCUSSION THREAD

**Discussion by jonadis** [mrlt8/docker-wyze-bridge#668] [General]
Curious if anyone knows if Wyze changed their plumbing/etc that caused LAN mode not to work anymore...
By happenstance I was looking at logs and saw:
```
WARNING: Camera is connected via RELAY mode. Stream may consume additional bandwidth!
```
And I don't remember ever seeing that before.
I went in and added the environment variable to choose LAN mode and now all I get is:
```
Connected via NON-LAN MODE! Reconnecting
```
over and over

I do limit the Wyze Cam's outbound network access to only what seemed to be needed, does anyone know if they started using different outbound ports or anything like that?

**Comment by mrlt8** [comment]
LAN mode still working for me. Are any of the cams on a beta firmware?

Can you do `docker-compose down` && `docker-compose up` to see if that helps?

**Reply by jonadis** [reply]
No to the beta firmware, and I had rebuilt the containers a few times already...
I had a thought to check the WatchGuard logs yesterday for the cameras and didn't see anything.
Today I had a thought to check the WatchGuard logs for the container and I saw some denies on port 10001, I just allowed that and we're back to good on LAN mode.
